src/can.rs

```diff
@@ -1,22 +1,33 @@
-use super::pac;
-use super::pac::can::TX;
+use bxcan::{FilterOwner, Instance, RegisterBlock};
+
 use crate::gpio::gpioa::{PA11, PA12};
 use crate::gpio::gpiob::{PB8, PB9};
 use crate::gpio::{Alternate, AF4};
+use crate::pac::CAN;
 use crate::rcc::Rcc;
 
-pub trait RxPin {}
-pub trait TxPin {}
+mod sealed {
```

Comment:
This disallowes the users of the crate to implement the RxPin themselves, right?

Reply:
Yeah